software/eonvelope.yml

```
@@ -0,0 +1,15 @@
+name: "Eonvelope"
+website_url: "https:/dacid99.gitlab.io/eonvelope"
+source_code_url: "https://gitlab.com/dacid99/eonvelope"
+description: "Preserve your emails with this open-source email archive designed for the self-hosted community"
```

```suggestion
description: "Preserve your emails with this open-source email archive designed for the self-hosted community."
```
Adding a dot at the end